uat/custom-components/client-python-paho/.gitignore

```diff
@@ -0,0 +1,7 @@
+src/dev-env
+*__pycache__*
+scripts/win_setup.bat

```

Comment:
That expose names of your local files which is not interested for public repo. You can move all scripts to scripts/local and exclude whole folder if really require these scripts to be local.

Reply:
fixed.
Keep "mqtt_client_control_pb2*" as it is output of gRPC generator